### PR TITLE
dev-main-readme

### PR DESCRIPTION
**Pull request by mdhillman**
This PR brings in an overhaul to the content of the top-level README file within the repository, hopefully bringing it more in line with other popular open source projects.

This includes a short recap of the recently added "Key features" sections (I felt using it verbatim would lead to an overly long document). However, the full text has been added to the main page of the Wiki, which is now linked to from the README.

Note that this PR will need @mk306-como's approval of the new TWA logo before it can be merged. Until this time, the PR will be marked as a draft.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,86 +1,63 @@
...
+The World Avatar project aims to create a digital ‘avatar’ of the real world. The digital world is composed of a dynamic knowledge graph (KG) that contains concepts and data that describe the world, and an ecosystem of autonomous computational agents that simulate the behaviour of the world and that update the concepts and data so that the digital world remains current in time. A knowledge graph is a network of data expressed as a directed graph, where the nodes of the graph are concepts or their instances (data items) and the edges of the graph are links between related concepts or instances. Knowledge graphs are often built using the principles of Linked Data. They provide a powerful means to host, query and traverse data, and to find and retrieve related information
```
Comment (sm453) [inline README.md:14]: Please re-add the full stop. We do need it. ;-)

File: README.md [hunk 2]
```diff
@@ -1,86 +1,63 @@
...
+**Digital Twin Visualisation Framework:**<br/>
+The [DTVF](./web/digital-twin-vis-framework) is a container that provides a web-page with which a user can view and explore geospatial information held in the TWA ecosystem. This includes not only map data, 3D-representations of buildings, and building information models (BIM), but also any data or information connected to them within the KG, such as data calculated by models, acquired sensor data, or other time series data.
 
-This library facilitates dealing with time series, which are ubiquitous in real-world applications, thus minimising code repetition for developers. Ontological mark-up, in the form of triples in a triple store, is designed to be minimal, just containing basic high-level information such as names and types of quantities of which a time series is meant to be stored. The actual data, i.e. time stamps and associated values of the quantities, are being held in a relational database, thus maintaining favourable execution speed and storage space efficiency. The time-series client can be found [here](./JPS_BASE_LIB/src/main/java/uk/ac/cam/cares/jps/base/timeseries).
+**Intelligent Agents:**<br/>
```
Comment (sm453) [inline README.md:31]: The word "intelligent" in this context is loaded. I would go for something less controversial like "autonomous".

File: README.md [hunk 3]
```diff
@@ -1,86 +1,63 @@
...
 
 The World Avatar represents information in a dynamic knowledge graph using technologies from the Semantic Web stack. Unlike a traditional database, the World Avatar contains an ecosystem of autonomous computational agents that continuously update it. The agents are described ontologically as part of the knowledge graph, and are able to perform actions on both concepts and instances. This design enables agents to update and restructure the knowledge graph, and allows them to discover and compose other agents simply by reading from and writing to the knowledge graph.
 
-## Key technical features ##
-
-Here are a few pointers on where to start:
+## Key Features ##
 
-### The stack
+Listed below are a number of the key technical features available within The World Avatar ecosystem. More information on these, and other features, can be see on [The World Avatar Wiki](https://github.com/cambridge-cares/TheWorldAvatar/wiki).
```
Comment (sm453) [inline README.md:20]: "can be seen"

**Review by sm453 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (mdhillman) [reply README.md]: Addressed in new commit.

**Comment by mdhillman** [comment]
Given my history with typos, it _might_ be worthwhile seeing if there's an off-the-shelf spell/grammar checking (with a configurable dictionary) action we can automate.

**Comment by sm453** [comment]
> Note that this PR will need @mk306-como's approval of the new TWA logo before it can be merged. Until this time, the PR will be marked as a draft.

Having discussed this amongst a few of us, we believe the most sensible way forward may be to just remove the logo from this PR and merge it, given that it is highly uncertain when/if we get feedback on the logo.

**Comment by sm453** [comment]
Change of plan: @mk306-como just approved the logo! :-)

**Review by sm453 — APPROVED** [review]

**Pull request merged**